examples/adjoints/adjoints_ex6/femparameters.h

```diff
@@ -0,0 +1,149 @@
+
+#ifndef __fem_parameters_h__
+#define __fem_parameters_h__
+
+#include "libmesh/libmesh_common.h"
+#include "libmesh/dof_map.h"
+#include "libmesh/enum_norm_type.h"
+#include "libmesh/function_base.h"
+#include "libmesh/getpot.h"
+#include "libmesh/id_types.h"
+#include "libmesh/parallel_object.h"
+#include "libmesh/periodic_boundaries.h"
+#include "libmesh/periodic_boundary.h"
+
+#include <limits>
+#include <map>
+#include <string>
+#include <vector>
+
+
+
+class FEMParameters : public libMesh::ParallelObject
```

Comment:
@roystgnr is the idea to have one copy of femparameters.h that is shared by several examples (maybe symbolic links)? Seems like we just keep copying this file from example to example, but I just checked and there aren't actually any differences between them...